devnet/docker-compose.yml

```diff
@@ -1,17 +1,10 @@
 version: "3.4"
 
 services:
-  xinfinnetwork1:
-    build:
-        context: .
-        dockerfile: ./Dockerfile
+  xinfinnetwork:
+    image: ${RPC_IMAGE}
```

Comment:
Can you create another compose file for all the network, current docker compose file is widely used by other members, and if we change the format, we need to manage all the conversation and way of deployment things from their end.